README.md

```diff
@@ -52,11 +43,20 @@ def fib(n: int) -> Iterator[int]:
         yield a
         a, b = b, a + b
 ```
+See [the documentation](http://mypy.readthedocs.io/en/stable/introduction.html) for more examples.
+
+For Python 2.7, the standard annotations are written as comments:
+```python
+def reverse(s):
+    # type: str -> str
```

Comment:
This should be ``# type: (str) -> str`` (note parentheses), otherwise it is ``syntax error in type comment``.